### ISSUE TITLE
Trying to install as R package throws warning about CRUX not being available

### ISSUE THREAD

**Issue by selkamand** [CCICB/CRUX#78]
```
Warning: namespace 'CRUX' is not available and has been replaced
by .GlobalEnv when processing object 'external_tool_metadata
```

**Comment by selkamand** [comment]
R CMD CHECK note

**Comment by selkamand** [comment]
Warning in gzfile(file, "rb") :
  cannot open compressed file '', probable reason 'No such file or directory'
Error in gzfile(file, "rb") : cannot open the connection
Warning: namespace ‘CRUX’ is not available and has been replaced
by .GlobalEnv when processing object ‘external_tool_metadata’
Warning in gzfile(file, "rb") :
  cannot open compressed file '', probable reason 'No such file or directory'
Error in gzfile(file, "rb") : cannot open the connection
Warning: namespace ‘CRUX’ is not available and has been replaced
by .GlobalEnv when processing object ‘external_tool_metadata’
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (CRUX)

**Comment by selkamand** [comment]
Solved in #93
